justfile

```diff
@@ -0,0 +1,25 @@
+alias b := build
+alias c := check
+alias f := fmt
+alias t := test
+
+_default:
+   @just --list
+
+# Build the project
+build:
+   cargo build
+
+# Check code: formatting, compilation, linting, and commit signature
+check:
+   cargo +nightly fmt --all -- --check
+   cargo check --all-features --all-targets
+   cargo clippy --all-features --all-targets -- -D warnings
+
+# Format all code
+fmt:
+   cargo +nightly fmt
```

Comment:
I don't use `rustup` and this would fail.
Can you please remove the `+nightly bound`

Reply:
Should also be
```suggestion
   cargo fmt --all
```